docs/available_commands.md

```diff
@@ -24,6 +24,7 @@ Currently, the command adds or modifies the following pieces of information:
 * Generates a *bom-ref* for components which don't have one, yet. The *bom-ref* will be a GUID.
 * If the path to a folder with license text files is provided, the text will be included in the SBOM, if the license has the corresponding `name`.
 * If a `license.name` is similar to an SPDX-ID, it will be replaced, e.g. `{"license": {"name": "The Apache License, Version 2.0"}}` leads to `{"license": {"id": "Apache-2.0"}}`. For this purpose a [JSON-file](https://github.com/Festo-se/cyclonedx-editor-validator/blob/main/cdxev/amend/license_name_spdx_id_map.json) is used, where we provide a mapping of license names to SPDX-IDs, based on this [license-mapping](https://github.com/CycloneDX/cyclonedx-core-java/blob/master/src/main/resources/license-mapping.json).
+* If the sbom contains a license which `name` includes any variation of the letter sequence  'unknown' and no or an empty `text`, the license will be removed.
```

Comment:
The condition should be just `name` including `unknown`. The text or URL can be gibberish that won't help you, as seen [here](https://github.com/CycloneDX/cyclonedx-dotnet/pull/755/files#diff-20191554df9f565da11a0cec8635bb3b29ca2e4930324725baeb9c0df6174d58R200)

Reply:
We agreed on leaving text as a failsafe, in case of some proprietary license with a bad naming sense.